docs/data/material/guides/understand-mui-packages/understand-mui-packages.md

```diff
@@ -129,3 +85,38 @@ export default function CustomSwitch() {
   return <SwitchUnstyled component={Root} {...label} />;
 }
 ```
+
+## Styling
+
+### Styled engines
+
+Styled engines come in two packages:
+
+- `@mui/styled-engine`: an [emotion](https://emotion.sh/docs/styled) wrapper.
+- `@mui/styled-engine-sc`: a [styled-components](https://styled-components.com/docs/basics#getting-started) wrapper.
+
+They were introduced in MUI Core's v5 to unlock more styling possibilities and enhanced customization features.
+It refers specifically to stylesheet generation (CSS-in-JS).
+
+These adapters unify the APIs of both `emotion` and `styled-components`, and you can choose to use whichever suits you best.
+The previous style library `@mui/styles` (JSS wrapper) is deprecated and will be removed in the future.
+
+In any case, it's likely that you won't interact much with these as they're used internally in `@mui/system`.
+Check [the RFC](https://github.com/mui/material-ui/issues/22342) to learn more about v5's styling solution change.
+
+### MUI System
+
+MUI System is a collection of CSS utilities to help you rapidly lay out custom designs.
+It uses the emotion adapter (`@mui/styled-engine`) as the default styled-engine to create the CSS utilities.
+
+Using MUI system:
+
+- You have full control of the `theme` object.
+- You can use `sx` prop normally as the `styled` API supports it by default.
+- You can have themeable components by using `styled` via slots and variants.
+
+> **Note**: To use it, you will have to install either `emotion` or `styled-components`, because the respective styled-engine package depends on it.
```

Comment:
```suggestion
:::warning
To use MUI System, you must install either emotion or styled-components, because the respective `styled-engine` package depends on it.
:::
```